htdocs/core/modules/commande/doc/pdf_einstein.modules.php

```diff
@@ -1408,14 +1408,16 @@ protected function _pagehead(&$pdf, $object, $showaddress, $outputlangs, $output
 			$hautcadre = 40;
 
 			// Show sender frame
-			$pdf->SetTextColor(0, 0, 0);
-			$pdf->SetFont('', '', $default_font_size - 2);
-			$pdf->SetXY($posx, $posy - 5);
-			$pdf->MultiCell(80, 5, $outputlangs->transnoentities("BillFrom"), 0, $ltrdirection);
-			$pdf->SetXY($posx, $posy);
-			$pdf->SetFillColor(230, 230, 230);
-			$pdf->MultiCell(82, $hautcadre, "", 0, 'R', 1);
-			$pdf->SetTextColor(0, 0, 60);
+			if (empty($conf->global->MAIN_PDF_NO_SENDER_FRAME)) {
```

Comment:
Not that einstein is deprecated and must be replace with eratosthene. Can you add it also into eratosthene ?

Reply:
@eldy it's done